account_invoice_supplier_ref_unique/tests/test_account_invoice_supplier_ref_unique.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016 Acsone
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import SavepointCase
+
+
+class TestAccountInvoiceSupplierRefUnique(SavepointCase):
+
+    @classmethod
+    def setUpClass(self):
+        super(TestAccountInvoiceSupplierRefUnique, self).setUpClass()
```

Comment:
Small points:
- rename `self` to `cls`
- `super()` parameters could be removed
- very extra: `tracking_disable` context key could be added to `env`

But not blocking